logging/logging.go

```diff
@@ -0,0 +1,20 @@
+package logging
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+	"go.elastic.co/ecszerolog"
+)
+
+// GetDefaultLogger generates and returns a default logger instance.
+func GetDefaultLogger() *zerolog.Logger {
+	logger := ecszerolog.New(os.Stdout, ecszerolog.Level(zerolog.DebugLevel)).
+		With().
+		Timestamp().
+		Caller().
```

Comment:
You can display stack trace of an error if add `Stack()`.

Reply:
Do we want stack trace?

Reply:
Stack tracę need error wrap, we do not want to do it for now. Maybe in future there will be better way to display stack trace, now we will use Origin